treescript/src/treescript/merges.py

```diff
@@ -60,7 +61,7 @@ def touch_clobber_file(config, repo_path):
             line = line.strip()
             if line.startswith("#") or line == "":
                 new_contents += f"{line}\n"
-        new_contents += "Merge day clobber"
+        new_contents += "Merge day clobber " + str(date.today())
```

Comment:
Looks good!

We prefer f-strings these days, though we are string-concatenating already in these two lines.

```python
        new_contents += f"Merge day clobber {str(date.today())}"
```

or

```python
        new_contents = f"{new_contents}Merge day clobber {str(date.today())}"
```

to fully get rid of the string concatenation.